bang/models.py

```diff
@@ -708,6 +709,34 @@ def __unicode__(self):
                     else self.name) if self.name or self.japanese_name else ''),
             )
         return u''
+    
+    cameo_members = models.ManyToManyField(Member, related_name='cameo_members', verbose_name=_('Other members in this card'))
+    _cache_cameos_blob = models.TextField(null=True)
```

Comment:
I made a thing for cache that makes that json thing for you which I guess would make sense to use here. Can you take a look?

In the future I plan to move all the manual cache thingies to use that instead of separate cached fields.

I didn't document it yet because I just made it, so I'll try to summarize it. Sorry about that 🙇 

- Requirements:
    - `_cache_XXX_days = 20`
    - `_cache_XXX_last_update = models.DateTimeField(null=True)`
    - `_cache_j_XXX = models.TextField(null=True)`
    - `to_cache_XXX` method which will get the item(s) from the db and should return a dictionary or a list of dictionaries.
- With that, you will get the following helpers:
    - `cached_XXX` property which will return a dictionary or a list of dictionary which contain what has been stored in the db using `to_cache_XXX` + more helpers that get added automatically:
        - `unicode` (name or id or '?')
        - If the dictionary(ies) returned by `to_cache_XXX` contain an `id`: `pk`, `item_url`, `ajax_item_url`, `full_item_url`, `http_item_url`
         - If the dictionary(ies) contain `image`: `image_url`, `http_image_url`
    - Methods `update_cache` and `force_update_cache` which both take the XXX as an argument (example: `card.update_cache('gachas')`)
- Optional:
    - `_cached_XXX_collection_name` will be used to format the auto URLs like `.item_url`, `.ajax_item_url`. If it's not provided, `XXX` will be used as the base URL.
    - `cached_XXX_pre` and `cached_XXX_extra` are both optional ***class*** methods which can be called when `cache_XXX` property is being retrieved. It's generally recommended to use `_extra` except to provide `unicode`, `id` or `image` dynamically.
    - If you're saving `i_` fields in the dictionary(ies) returned by `to_cache_XXX`, you should provide `_cache_XXX_fk_class` which will be used when calling `cache_XXX` to return the usual helpers for `i_` fields. (example: `_cache_gachas_fk_class = Gacha` or _`cache_gachas_fk_class = classmethod(lambda _s: Gacha)`)

Examples:
-  [Gacha and events in BanGDream card model](https://github.com/MagiCircles/BanGDream/blob/master/bang/models.py#L617)
- [Unit tests which show some examples of returned caches](https://github.com/MagiCircles/MagiCircles/blob/master/tests/test/test_item_model_cache.py) (see also [models used for the unit tests](https://github.com/MagiCircles/MagiCircles/blob/master/tests/test/models.py#L149) (Idol, Card, Gacha))